nntrainer/include/tensor.h

```diff
@@ -84,7 +91,7 @@ class Tensor {
    * @param[in] h height location
    * @param[in] w width location
    */
-  float getValue(int batch, int h, int w);
+  float getValue(unsigned int batch, unsigned int h, unsigned int w);
```

Comment:
Are you limiting the dimension to be h x w x batch?

Reply:
There is another PR for that. Thanks.